packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.tsx

```diff
@@ -55,67 +61,125 @@ const CompleteResetPassword = ({
+        recover it.
+      */}
+      {hasConfirmedRecoveryKey === false &&
+        recoveryKeyExists &&
+        hasSyncDevices && (
+          <div
+            className={`bg-red-100 rounded-lg text-sm mt-6 text-left rtl:text-right p-3`}
+            data-testid="warning-message-container"
+          >
+            <div className="flex">
+              <BangIcon
+                role="img"
+                className="flex-initial mr-2 rtl:ml-2 mt-1"
+              />
+              <div className="flex-1 text-xs">
+                <FtlMsg id="password-reset-could-not-determine-account-recovery-key">
+                  Have an account recovery key?
+                </FtlMsg>{' '}
+                <br />
                 <Link
                   to={`/account_recovery_confirm_key${location.search}`}
                   state={locationState}
-                  className="link-white underline-offset-4"
+                  className="link-blue"
                   onClick={() =>
                     GleanMetrics.passwordReset.createNewRecoveryKeyMessageClick()
                   }
                 >
-                  Reset your password with your account recovery key.
-                </Link>
-              </FtlMsg>
-            </>
-          </Banner>
+                  <FtlMsg id="password-reset-use-account-recovery-key">
+                    Reset your password with your recovery key.
+                  </FtlMsg>
+                </Link>{' '}
+              </div>
+            </div>
+          </div>
         )}
 
-      {errorMessage && <Banner type={BannerType.error}>{errorMessage}</Banner>}
-
-      {hasConfirmedRecoveryKey ? (
-        <FtlMsg id="account-restored-success-message">
-          <p className="text-sm mb-4">
-            You have successfully restored your account using your account
-            recovery key. Create a new password to secure your data, and store
-            it in a safe location.
-          </p>
-        </FtlMsg>
-      ) : (
-        <WarningMessage
-          warningMessageFtlId="complete-reset-password-warning-message-2"
-          warningType="Remember:"
-        >
-          When you reset your password, you reset your account. You may lose
-          some of your personal information (including history, bookmarks, and
-          passwords). That’s because we encrypt your data with your password to
-          protect your privacy. You’ll still keep any subscriptions you may have
-          and Pocket data will not be affected.
-        </WarningMessage>
-      )}
-
-      {/* Hidden email field is to allow Fx password manager
-          to correctly save the updated password. Without it,
-          the password manager tries to save the old password
-          as the username. */}
+      {hasConfirmedRecoveryKey === false &&
+        recoveryKeyExists === false &&
+        hasSyncDevices && (
+          <div
+            className={`bg-orange-50 rounded-lg text-sm mt-6 text-left rtl:text-right p-3`}
```

Comment:
```suggestion
            className={`bg-orange-50 rounded-lg text-sm mt-6 text-left rtl:text-right p-4`}
```

Suggest increasing the padding here and removing the bottom padding on the bullets section to balance out the padding.